packages/strapi-plugin-content-type-builder/controllers/ContentTypes.js

```diff
@@ -62,10 +64,15 @@ module.exports = {
         components: body.components,
       });
 
+      const metricsProperties = {
+        kind: contentType.kind,
+        hasDraftAndPublish: hasDraftAndPublish(contentType.schema),
+      };
+
       if (_.isEmpty(strapi.api)) {
-        await strapi.telemetry.send('didCreateFirstContentType', { kind: contentType.kind });
+        await strapi.telemetry.send('didCreateFirstContentType', metricsProperties);
       } else {
-        await strapi.telemetry.send('didCreateContentType', { kind: contentType.kind });
+        await strapi.telemetry.send('didCreateContentType', metricsProperties);
```

Comment:
Should we also send it when the user update a content-type or something like that ?
As d&P is enabled by default, maybe people forget to disable it and do it only after.

Reply:
we don't have an event for that :)

Reply:
Oki, @sam-pires are you aware of this?

Reply:
yep